readme.md

```diff
@@ -104,7 +107,9 @@ Your Answer:
 
 Write some basic CSS that demonstrates changing a CSS property when the device width drops below 40rem.
 
-```css
+```body{
+    @media (max-width: 40rem) background: green;
+}
 
 ```
```

Comment:
`@media` should be placed outside of the `body` selector. We only want those changes to be applied to `body` when the width is at or below `40rem`. So it should look something like this...

```js
@media (max-width: 40rem) {
  body{
    background: lemonchiffon;
  }
}
```